README.md

```diff
@@ -29,6 +29,15 @@ any web server to check it out in your browser:
 
 The site is now available at <http://0.0.0.0:8000>.
 
+If you need to preview the site from another machine, pass the host's address via `--bind`. For example, if running on 192.168.72.73:
```

Comment:
Huh. I would have expected this to work with 0.0.0.0...

Reply:
Are you saying I should try doing `python3 -m http.server --bind 0.0.0.0` ?

/me tries it.

Okay, you're right, that works too.

(The crucial thing is that it *doesn't* work if you don't pass `--bind` at all)